tests/test_translate.py

```diff
@@ -63,11 +63,14 @@ def sample_wherefail(
     failure_stride,
     test_name,
     ignore_near_zero_errors,
+    near_zero,
     xy_indices=False,
 ):
     found_indices = np.where(
         np.logical_not(
-            success_array(computed_data, ref_data, eps, ignore_near_zero_errors)
+            success_array(
+                computed_data, ref_data, eps, ignore_near_zero_errors, near_zero
```

Comment:
Looks like there is one other call to success_array in this method that needs the near_zero argument (it is an optional test argument that prints out the errors in a nice grid like layout), under xy_indices

Reply:
Nice catch!